ResearchKit/Common/ORKHelpers.m

```diff
@@ -395,6 +395,84 @@ CGFloat ORKTableViewLeftMargin(UITableView *tableView) {
     return font;
 }
 
+NSURL *ORKURLFromBookmarkData(NSData *data) {
+    if (data == nil) {
+        return nil;
+    }
+    
+    BOOL bookmarkIsStale = NO;
+    NSError *bookmarkError = nil;
+    NSURL *bookmarkURL = [NSURL URLByResolvingBookmarkData:data
+                                                   options:NSURLBookmarkResolutionWithoutUI
+                                             relativeToURL:nil
+                                       bookmarkDataIsStale:&bookmarkIsStale
+                                                     error:&bookmarkError];
+    if (! bookmarkURL) {
+        ORK_Log_Debug(@"Error loading URL from bookmark: %@", bookmarkError);
```

Comment:
`ORK_Log_Debug()` seems a bit overused currently: it's used elsewhere to log non-problemating behaviour like `viewDidAppear`, `dealloc`, etc. There are some potentially serious errors, like this one, which may benefit from a stronger logging mechanism.

Should we maybe replace `ORK_Log_Debug()` and `ORK_Log_Oops()` by `ORK_Log_Debug_Info()`, `ORK_Log_Debug_Warn()`, `ORK_Log_Debug_Error()` (and provide a way of configuring log verbosity)?

There are also things like [CocoaLumberjack] (https://github.com/CocoaLumberjack/CocoaLumberjack), but those may be going overboard.